### ISSUE TITLE
Task list for VK_EXT_shader_atomic_float

### ISSUE THREAD

**Issue by vkushwaha-nv** [KhronosGroup/Vulkan-Docs#1323]
The Vulkan 1.2.148 spec release includes the [VK_EXT_shader_atomic_float](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#VK_EXT_shader_atomic_float) extension. This extension allows a shader to contain floating-point atomic operations on buffer, workgroup, and image memory.

The task list for the VK_EXT_shader_atomic_float extension release is:

* [x]   Vulkan specification (https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#VK_EXT_shader_atomic_float)
* [x]   SPIRV specification (https://github.com/KhronosGroup/SPIRV-Registry/pull/77)
* [x]   SPIRV headers change
* [x]   SPIRV tools
* [x]   Glslang
* [x]   GLSL extension (https://github.com/KhronosGroup/GLSL/pull/129)
* [x]   Validation layer
* [x]   Conformance tests

As each component is made public, the task will be checked off. When all tasks have been completed this issue will be closed and the extension will be fully released.

**Comment by marty-johnson59** [comment]
Doing some year-end house-cleaning on open public issues...  @vkushwaha-nv, is this one waiting on validation layer?  If so, is there an issue/owner for that?  Thanks,

Marty

**Comment by vkushwaha-nv** [comment]
Yes, I don't think there is an issue/owner. I am not sure who can work on that.

**Comment by marty-johnson59** [comment]
OK, perhaps we need a new issue.  @TomOlson to advise as I'm not on who should own this one.  Thanks
